test/src/scene/textureTest.cpp

```diff
@@ -0,0 +1,152 @@
+#include <gtest/gtest.h>
+#include "utils.hpp"
+#include "scenes.hpp"
+#include "lidars.hpp"
+
+#ifdef RGL_BUILD_ROS2_EXTENSION
+#include <rgl/api/extensions/ros2.h>
+#endif
+
+constexpr unsigned short UsMaxValue = 255;
```

Comment:
`std::numeric_limits<unsigned short>::max()` should work (with `#include <limits>`)

Reply:
Discussed offline